share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -1,6 +1,58 @@
 # Formula Cookbook
 A formula is a package definition written in Ruby. It can be created with `brew create $URL` and installed with `brew install $FORMULA` and debugged with `brew install --debug --verbose $FORMULA`. Formulae use the [Formula API](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula) which provides various Homebrew-specific helpers.
 
+Table of Contents
+=================
```

Comment:
Do you mind using `## Table of Contents` to be consistent with the existing style? Thanks!